components/eam/src/physics/crm/samxx/test/shr_const_mod.F90

```diff
@@ -18,7 +18,7 @@ MODULE shr_const_mod
    real(R8),parameter :: SHR_CONST_REARTH  = 6.37122e6_R8    ! radius of earth ~ m
    real(R8),parameter :: SHR_CONST_G       = 9.80616_R8      ! acceleration of gravity ~ m/s^2
 
-   real(R8),parameter :: SHR_CONST_STEBOL  = 5.67e-8_R8      ! Stefan-Boltzmann constant ~ W/m^2/K^4
+   real(R8),parameter :: SHR_CONST_STEBOL  = 5.670374419e-8_R8      ! Stefan-Boltzmann constant ~ W/m^2/K^4
```

Comment:
Instead of setting the value explicitly, use the shr_const_mod module and set it to SHR_CONST_STEBOL.